components/brave_new_tab_ui/components/default/footer/togetherTooltip/button.tsx

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2020 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+import styled from 'brave-ui/theme'
+
+const Button = styled('button')`
```

Comment:
Apparently this is a new design system-able button style that we can make more shared when the design team iterates a bit more